.travis.yml

```diff
@@ -9,4 +9,5 @@ before_install:
   - make vendor
 
 script:
-  - make check test
+  - make check test install
```

Comment:
No need to add install here, test already does that as the binaries are needed to run the bats tests.